cmd/ocm/create/cluster/cmd.go

```diff
@@ -668,6 +689,17 @@ func run(cmd *cobra.Command, argv []string) error {
 		args.subscriptionType = parseSubscriptionType(args.subscriptionType)
 	}
 
+	// If marketplace-gcp subscription type is used, flag marketplace-gcp-terms must be set to true.
+	gcpBillingModel, _ := billing.GetBillingModel(connection, billing.MarketplaceGcpSubscriptionType)
+	gcpSubscriptionTypeTemplate := parseSubscriptionType(subscriptionTypeOption(gcpBillingModel.ID(),
+		gcpBillingModel.Description()).Value)
+	isGcpMarketplaceSubscriptionType := args.subscriptionType == gcpSubscriptionTypeTemplate
+	if isGcpMarketplaceSubscriptionType {
```

Comment:
nit: isGcpMarketplaceSubscriptionType is a long name for a variable. I would suggest using `isGcpMarketplace`.